src/lib/tasks/server/execute_server_query_task.cpp

```diff
@@ -9,7 +9,7 @@ void ExecuteServerQueryTask::_on_execute() {
     _sql_pipeline->get_result_table();
     // Indicate that execution is finished. The result is accessed from outside so we need an empty promise.
     _promise.set_value();
-  } catch (const std::exception& exception) {
+  } catch (...) {
```

Comment:
In ` src/lib/tasks/server/create_pipeline_task.cpp`, you've been doing this:
```
-  } catch (const std::exception& exception) {
+  } catch (const std::exception&) {
```

Here, you're using `catch(...)`. Since you access the current exception in both cases, I am wondering if this was done on purpose. Assuming both are valid now with the stricter checking and both are semantically identifical: which one should we use in Opossum?

Reply:
Good point. `std::exception` does not cover exceptions that do not inherit from `std::exception`, so `...` is better. I fixed this part where exceptions were used as a regular control method.